guide/getting-started/user-research.md

```diff
@@ -0,0 +1,191 @@
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/user-research/user-research.jpg"
+   retina = "/assets/images/guide/getting-started/user-research/user-research@2x.jpg"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+**In short, user research helps teams create better products in a more focused and efficient manner.**
+
+Prioritizing on the features and improvements that make the biggest difference is crucial. What exactly those are is often a matter of heavy debate with clashing opinions and rationale. A great way to get past individual opinions is to make data-informed decisions. User research creates data from the start and end points of our efforts. Products are created because there are specific unaddressed needs, then they go through development and finally back in the hands of the target audience for use and evaluation. Many times we are too occupied with the complexities of development, but the real measure of success is whether user needs are satisfied well.
+
+</div>
+
+User research allows us to effectively connect these dots and highlight the quickest path between them. This results in more appropriate product roadmaps, more focused team efforts, and happier users. This is especially important for projects with limited resources, which many open-source efforts are.
+
+## Unique aspects of user research in Bitcoin
+
+There are many ways in which research and development are similar to other digital products. As to not repeat techniques and approaches that have been well-documented elsewhere, let's look at some areas in which Bitcoin may be different and unique.
+
+### Global audiences and use cases
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/user-research/global-audience.jpg"
+   retina = "/assets/images/guide/getting-started/user-research/global-audience@2x.jpg"
+   alt-text = ""
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+Most digital products start with a specific country, culture, language or legal framework in mind. This is especially true for financial applications, which are typically highly regulated and need to slowly and strategically evolve. While Bitcoin did start with a single language, its use case was not tied to any particular context, and this has resulted in a much more diverse user base from the start.
+
+This has resulted in diverse and lively online commmunities that are deeply invested in bitcoin. Designers and researchers have vast amounts of public conversation at their hands that allow them to better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+
+Some of the following links have been around for a long day and provide a huge archive and history of information and conversation around bitcoin:
```

Comment:
Suggest changing to '...for a long time...' instead of '...for a long day...'

Reply:
Fixed, thank you.